### ISSUE TITLE
[Suggestion]: Mass linking of devices, quality-of-life benefit

### ISSUE THREAD

**Issue by Whackjob** [tsunamayo/Starship-EVO#2976]
### Short Description of the issue:

Linking a large amount of items to a specific parent object takes too long, going one-at-a-time.  So here's what I'd like to suggest:  Right click with the link tool to set the parent, and left click to link the parent to the child.  That way you can say right click and hook a turret tile, and then simply left click to link weapon barrels to it.

### Steps to reproduce the issue:

Put a lot of things and then have to link it.

### Environment
- Starship EVO version:  Current experimental
- Graphics Card:  Quadro P5000

### Additional information:

Pleaz

**Comment by 5URG3** [comment]
Yes, I absolutely agree. Setting up even a small number of turrets, weapons or landing gear is quite tedious, especially on larger builds. Anything to speed that up would be welcome.

Whackjob's RMB/LMB setup seems like a good idea to me.

**Comment by cloudrambler** [comment]
Agree with the OP proposal. However a workaround for pure logic features would be to place a logic gate near the group of entities. That can be linked to the source once and then it will be faster to link it to multiple local entities. Probably won't work with auto turrets.

**Comment by tsunamayo** [comment]
Yes it is a valid suggestion, we already had a discussion about that (cant find the thread). On the to-do list!

**Comment by Claakin** [comment]
Nice! Thanks!
